### PR TITLE
Etcd cleanup

### LINKED ISSUE #5063: etcd slowly increase in size, sessions cleanup missing
Sensu store sessions inside etcd but does not clean up. This will slowly increase the size of etcd, potentially stopping etcd as there is a hard limit at 2GB (default).

## Expected Behavior
Clean up old/expired sessions

## Current Behavior
Old sessions are kept under `/sensu.io/user_sessions/admin/[uuid]` inside etcd.

You can see the number of sessions kept in etcd:
`etcdctl [insert authentication and endpoint info] --command-timeout=200s --dial-timeout=200s get "/sensu.io/user_sessions/admin" --prefix=true --keys-only | grep -c -v -e '^\s*$' `. After a few month we were getting close to 1.7GB, with nearly 5.000.000 keys in the etcd, and over 4.9m just being sessions.

## Possible Solution
Quick fix is to delete all sessions:
`etcdctl [insert authentication and endpoint info] --command-timeout=200s --dial-timeout=200s del "/sensu.io/user_sessions/admin" --prefix=true`
This reduced etcd from 1.7GB to 392MB.

Proper fix would be to clean expired sessions.

## Steps to Reproduce (for bugs)
The scale of the issue would probably hard to reproduce but we can see that the sessions are kept in etcd and keeps increasing.

## Your Environment
Sensu 6.10
Ubuntu 22.04 LTS

### PR DESCRIPTION
**Pull request by ManishaKumari295**
closed #5063

**Description**
The user sessions gets piled up in etcd occupying upto 2bg space .

**Change in behavior**
Deleting the etcd user_sesssions them after 6 minutes of creation ( Access token of session gets expired in 5 mnts) This change will make sure the memory occupancy uptil 1.8 GB will be eradicated

**Added**
Added TTL time of 6 minutes in session.go file for each entry regarding user-sessions.

**Fixed**
This fixes the etcd space occupancy issue .

**Change verification**
Do sensuctl configure.
Check respective entry in etcd ar /sensu.io/user-sessions/admin/xxyyzz[session_id]
this entry will be deleted automatically after 11 minutes if the backend keep .
I have the video of default behaviour ( which caused the issue) and the video of fixed behaviour with TTL=2mins , unfortunately cannot be uploaded due to big size

### CODE REVIEW

File: backend/store/etcd/session.go [hunk 1]
```diff
@@ -30,12 +30,18 @@ func (s *Store) GetSession(ctx context.Context, username, sessionID string) (str
 }
 
 // UpdateSession applies the supplied state to the session uniquely identified
-// by the given username and session ID.
+// by the given username and session ID with attached lease and TTl for each key
 func (s *Store) UpdateSession(ctx context.Context, username, sessionID, state string) error {
-	if _, err := s.client.Put(ctx, userSessionPath(username, sessionID), state); err != nil {
+
+	leaseDuration := jwt.DefaultAccessTokenLifespan
+	ttl := int64(leaseDuration.Minutes()+1) * 60
```
Comment (elfranne) [inline backend/store/etcd/session.go:37]: Any reason to adding a minute to the lease?

File: CHANGELOG-6.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic
 Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.11.1] - 2024-09-12
+
+### Changed
+- ADD TTL to each user session in the etcd data store to prevent leak.
+- TTl value is DefaultAccessTokenLifeSpan + 1 minute
```
Comment (fguimond) [inline CHANGELOG-6.md:13]: Just one comment per change is required. The second entry can be removed.

File: backend/store/etcd/session.go (hunk 1 above)
Comment (ManishaKumari295) [reply backend/store/etcd/session.go:37]: This is to keep a buffer in deletion, no other reason.

**Review by fguimond — COMMENTED** [review]
LGTM

**Comment by ManishaKumari295** [comment]
Merging to develop/6

**Pull request merged**